Tools/Scripts/hooks/prepare-commit-msg

```diff
@@ -91,22 +94,54 @@ def cherry_pick(content):
     cherry_picked = os.environ.get('GIT_WEBKIT_CHERRY_PICKED')
     bug = os.environ.get('GIT_WEBKIT_BUG')
 
+    if PREFER_RADAR and not bug:
+        RADAR_RES = [
+            re.compile(r'<?rdar://problem/(?P<id>\d+)>?'),
+            re.compile(r'<?radar://problem/(?P<id>\d+)>?'),
+            re.compile(r'<?rdar:\/\/(?P<id>\d+)>?'),
+            re.compile(r'<?radar:\/\/(?P<id>\d+)>?'),
+        ]
+        seen_empty = False
+        for line in content.splitlines():
+            if not line and seen_empty:
```

Comment:
Do we need to call `line.strip()` ?

Reply:
Maybe if we were being pedantic, I think it's probably fine for whitespace to count as a line with text, in this case. We're really just doing this to be speedy, changes which start breaking our expected commit message format are going to do weird things with bug attribution. Worth nothing this is also what `webkitscmpy`'s library code already does.